### ISSUE TITLE
Feature's Date/time definition on creation

### ISSUE THREAD

**Issue by flaviostutz** [opengeospatial/WFS_FES#231]
We have the "datetime" attribute in query params for filtering features but I couldn't find any references in API for creating Features with datetime info along with GeoJSON. What's the correct way?

If there is no "correct" way of doing this yet, I propose the WFS 3.0 spec to adopt the date/time structure for GeoJSON as defined by https://github.com/geojson/geojson-ld.
  * See discution here: https://github.com/geojson/geojson-ld/issues/9
  * See proposal here: https://gist.github.com/sgillies/11404768

Example:
```
{
  "geometry": {
    "coordinates": [
      0.0,
      0.0
    ],
    "type": "Point"
  },
  "id": "1",
  "properties": {"foo": "bar"},
  "type": "Feature",
  "when": {
    "datetime": "2014-04-24",
    "@type": "Instant"
  }
}
```

**Comment by flaviostutz** [comment]
@cholmes, what do you think about this? Are there any references on this on STAC?

**Comment by cportele** [comment]
Discussion 17 June: Concerns that this is just an old discussion (2014) and that it will not become a GeoJSON practice.

**Comment by cmheazel** [comment]
The general principle for JSON seems to be that whatever is not prohibited is allowed. So adding a "when" property to a GeoJSON document is allowed, even if we don't identify that property in the API spec.  Which leads us to two points:
1) Nothing in the current spec precludes an implementation from adding this property
2) The proper place to specify a standard "when" property is the GeoJSON spec.

**Comment by m-mohr** [comment]
In STAC, we have currently three date/time related to items:
* datetime - The searchable date and time of the assets/data
* updated - Date and time this metadata file was updated last. This is NOT the timestamp the asset was updated last.
* created - Creation date and time of this metadata file. This is NOT the timestamp the asset was created.

see https://github.com/radiantearth/stac-spec/blob/dev/item-spec/item-spec.md#properties-object

These are all stored in the properties. And we'd favor if we add such data in the properties, not on the top level.

**Comment by nmtoken** [comment]
By comparison in INSPIRE (ISO 191115) metadata, we have a single date for date the metadata was created/last updated (_MD_Metadata.dateStamp_).    We have temporal extent (_EX_Extent > EX_TemporalExtent.extent_) to delineate the  searchable date and time of the data,  and also we have dataset reference dates (_MD_Identification.citation > CI_Citation.date_) that record when the dataset (not metadata) was created, published, or revised

**Comment by cholmes** [comment]
I don't think it needs to be in the GeoJSON spec. I think it is valuable to have a few 'standard' time fields that exist in OGC land (or are recommended in OGC land and we reuse someone else's definitions). We've made a crack at it in STAC, but we're not fully set on them (though we do want to get to 1.0 relatively soon, at which time we will be set). 

They don't even need to be 'core' on WFS, but a small extension to the core that anyone wondering 'how should I describe my updated / created / etc timestamps' then we have some fields to recommend they use, instead of everyone coming up with their own. Indeed ideally in STAC we are just 'using' a main timestamp that is defined elsewhere. 

Happy to also try to align with ISO 19115 - STAC will try to follow whatever is recommended for WFS 3.

And yes, we prefer in properties, so the timestamps actually show up in all the geo software that understands geojson today.

**Comment by cportele** [comment]
Decision in Leuven meeting: No change to the core, it is up to implementations how to encode temporal properties in GeoJSON, typically as part of the "properties" member.

If pre-defined properties would be defined, this would be an extension (or a different spec).

**Comment by flaviostutz** [comment]
The lack of a minimum definition on this will create voids in the way time is expressed in WFS invocations, leading to incompatibilities regarding to time. When we are taking about lots of data, the time dimension is as important as the location itself because of the nature of sensed data.

For example, I have a simple implementation of WFS 3.0 that uses Postgis as backend (https://github.com/flaviostutz/wfsgis). On top of it, I created a MVT (Mapbox View Tile) server for fetching parts of data for huge datasets (https://github.com/flaviostutz/wfs-tiler) in visualization maps (https://github.com/flaviostutz/map-demos). All those tools supports a proprietary way of expressing datetime data in vectors, leading to incompatibilities of those "WFS compatible" tools with other (like a QGIS plugin for WFS 3.0, for example).

It seems to me a little inconsistent to have a "time" property for queries in the spec and no "datetime" spec for creating features, even a simple convention like "use property 'datetime' for expressing time in GeoJSON"
